pom.xml

```diff
@@ -423,7 +420,6 @@
       </plugin>
     </plugins>
   </reporting>
-
```

Comment:
See review comment above, one empty line here between `</reporting>` and `<profiles>` would be perfect.

Reply:
Thanks @lmarniazman . All comments are fixed.